R/Sound_DMR.R

```diff
@@ -844,11 +851,13 @@ add_changepoint_info <- function(whole_df, changepoint, col_name) {
     # Find the MethRegion, MethGroupName, and MethRegionLength
     whole_df[MethRegionName] = 0
     whole_df[MethRegionLengthName] = 0
+    whole_df[MethCytosinesName] = 0
     for(i in 1:nrow(BP_CG)){
       whole_df[c(BP_CG[i, 'start']:BP_CG[i, 'stop']),MethRegionName] <- BP_CG[i, 'DifMean']
       whole_df[c(BP_CG[i, 'start']:BP_CG[i, 'stop']),MethRegionLengthName] <- abs(as.numeric(BP_CG[i, 'BPStop']) -
                                                                                     as.numeric(BP_CG[i, 'BPStart']))
       whole_df[c(BP_CG[i, 'start']:BP_CG[i, 'stop']),MethGroupName] <- i
+      whole_df[c(BP_CG[i, 'start']:BP_CG[i, 'stop']),MethCytosinesName]<-BP_CG$stop[i]-BP_CG$start[i]+1
```

Comment:
Are these the number of cytosines per DMR? I know Melis was asking for that feature the other day. I think she was also interested in a column showing the maximum distance between two cytosines within a DMR